index.html

```diff
@@ -85,6 +84,8 @@ <h4>Island Getaway</h4>
   <footer class="footer">
     <p>Copyright Fun Bus 2018</p>
   </footer>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/2.1.2/TweenMax.min.js"></script>
+  <script src="js/index.js" async></script>
```

Comment:
for some reason i'm having trouble loading tweenmax when i open your project in the html preview. none of the animations are working for me ;(

 https://htmlpreview.github.io/?https://github.com/dhamano/DOM-II/blob/dustin-hamano/index.html

Reply:
yeah, it's weird. If you go into the inspector and mess with the url it'll suddenly load GSAP and the animation will work. however, since the tween breaks the deletion of the popup modal that doesn't work. or something